cmd/crypto/kes.go

```diff
@@ -103,10 +108,28 @@ func NewKes(cfg KesConfig) (KMS, error) {
 		return nil, err
 	}
 	return &kesService{
-		client: kes.NewClient(cfg.Endpoint, &tls.Config{
-			Certificates: []tls.Certificate{cert},
-			RootCAs:      certPool,
-		}),
+		client: &kesClient{
```

Comment:
you should just leave this to NewCustomHTTPTransport()

Reply:
I cannot use `NewCustomHTTPTransport()` since it's in `cmd` and `cmd` imports `cmd/crypto` such that we would create a cyclic dependency.

Reply:
You pass the transport in kesConfig @aead

Reply:
Okay, yeah - this will work...